### PR TITLE
[stable/prometheus-operator] Expose Prometheus PSP allowedCapabilitie…

### LINKED ISSUE #15091: [stable/prometheus-operator] Make allowedCapabilities in PSP configurable
**Is your feature request related to a problem? Please describe.**

I'd like to add a sidecar to the prometheus pod (vault agent - to provide vault token which can be used then by serviceMonitor). Problem is that it won't start with PSP from templates (https://github.com/helm/charts/blob/master/stable/prometheus-operator/templates/prometheus/psp.yaml) because it requires some additional capabilities.

**Describe the solution you'd like**

I'd like to be able to configure `allowedCapabilities` of the PSP in values file.
I can submit a PR, but first I'd like to get the feedback from maintainers if such a solution would be acceptable. Maybe more 'general' solution for modifying PSP should be added?

**Describe alternatives you've considered**
Patching PSP after chart installation.

### PR DESCRIPTION
**Pull request by mwieczorek**
…s in values file

Signed-off-by: mwieczorek <wieczorek-michal@wp.pl>

#### What this PR does / why we need it:
Described in #15091

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #15091

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/prometheus-operator/Chart.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ name: prometheus-operator
 sources:
   - https://github.com/coreos/prometheus-operator
   - https://coreos.com/operators/prometheus
-version: 5.16.0
+version: 5.16.1
```
Comment (vsliouniaev) [inline stable/prometheus-operator/Chart.yaml]: Because this is a feature, rather than a bugfix, you should bump the minor version in semver

**Review by vsliouniaev — CHANGES_REQUESTED** [review]

File: stable/prometheus-operator/Chart.yaml (hunk 1 above)
Comment (mwieczorek) [reply stable/prometheus-operator/Chart.yaml]: True, thanks, bumped the version

**Comment by vsliouniaev** [comment]
/lgtm
Thanks!

**Pull request merged**